tests/dream/io_test.py

```diff
@@ -0,0 +1,77 @@
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright (c) 2023 Scipp contributors (https://github.com/scipp)
+import pytest
+
+from ess import dream
+
+
+@pytest.fixture
+def filename():
+    return dream.data.get_path('DREAM_nexus_sorted-2023-12-07.nxs')
+
+
+@pytest.mark.filterwarnings("ignore:Failed to load /entry/instrument/monitor_bunker")
+@pytest.mark.filterwarnings("ignore:Failed to load /entry/instrument/monitor_cave")
+@pytest.mark.filterwarnings("ignore:Failed to load /entry/instrument/polarizer/rate")
+@pytest.mark.filterwarnings("ignore:Failed to load /entry/instrument/sans_detector")
```

Comment:
Are you planning to remove those filters when we have a complete file?

Reply:
I wanted to know if these are permanent and a problem with the loader or just temporary until we get better files.